src/screens/Vault/SignerAdvanceSettings.tsx

```diff
@@ -1177,6 +1213,24 @@ function SignerAdvanceSettings({ route }: any) {
         }}
         buttonCallback={() => setDetailModal(false)}
       />
+      <KeeperModal
+        visible={singleSigModal}
+        close={() => setSingleSigModal(false)}
+        title={signerTranslation.missingSingleSigTitle}
+        subTitle={
+          formatString(
+            signerTranslation.missingSingleSigSubTitle,
+            getSignerNameFromType(signer.type, signer.isMock, false).replace(/\*+/g, '')
+          ) as string
+        }
+        modalBackground={
+          privateTheme ? `${colorMode}.modalPrivateBackground` : `${colorMode}.modalWhiteBackground`
```

Comment:
Also this modalPrivateBackground color doesn't seem to be used anywhere else. Is it really needed?